CHANGELOG.md

```diff
@@ -8,11 +8,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Breaking changes
 
+* Updated titles of some categories. [#914](https://github.com/elastic/package-registry/pull/914)
```

Comment:
This shouldn't be a breaking change, but not sure if clients are using the titles for something apart of showing them. 

@kpollich do you know if the integrations UI uses the titles for something apart of to show them?

Reply:
I don't think so - seems like the custom integrations plugin relies on the category ID's for its logic, and Integrations UI just uses the title for disply purposes https://github.com/elastic/kibana/blob/2c774f536e2cd0b01b8829d842684e67df2c3d57/src/plugins/custom_integrations/common/index.ts